react/features/app/components/App.native.tsx

```diff
@@ -12,6 +12,7 @@ import { clientResized, setSafeAreaInsets } from '../../base/responsive-ui/actio
 import DimensionsDetector from '../../base/responsive-ui/components/DimensionsDetector.native';
 import { updateSettings } from '../../base/settings/actions';
 import JitsiThemePaperProvider from '../../base/ui/components/JitsiThemeProvider.native';
+import { isEmbedded } from '../../base/util/embedUtils.native';
```

Comment:
Do we need the .native suffix? It should work without it, isn't it?

Reply:
If it's a native file it's better to have it because it helps the auto-complete not get lost sometimes.